linux-anvil-comp7/Dockerfile

```diff
@@ -15,7 +15,6 @@ RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
 RUN yum update -y && \
     yum install -y \
         bzip2 \
-        make \
```

Comment:
Dropping make here doesn't work as make is a dependency of yum in x86_64 if I remember correctly

Reply:
Good point. Looking inside the `centos:6` image I see...

```
$ which make
/usr/bin/make
```